db/schema.rb

```diff
@@ -36,7 +35,6 @@
     t.string  "uid",                 limit: 255
     t.string  "organisation_slug",   limit: 255
     t.string  "permissions",         limit: 255
-    t.boolean "remotely_signed_out",             default: false
+    t.boolean "remotely_signed_out", default: false
   end
-
```

Comment:
I think `schema.rb` should probably be excluded from linting as it's auto-generated.